packager_test.go

```diff
@@ -123,16 +123,54 @@ func TestPackagerZKPPacker(t *testing.T) {
 	marshalledMsg, err := json.Marshal(msg)
 	assert.NoError(t, err)
 
-	envelope, err := pm.Pack(packers.MediaTypeZKPMessage, marshalledMsg, &senderID)
+	envelope, err := pm.Pack(packers.MediaTypeZKPMessage, marshalledMsg, packers.ZKPPackerParams{SenderID: &senderID})
 	assert.NoError(t, err)
 
 	unpackedMsg, unpackerType, err := pm.Unpack(envelope)
-	fmt.Printf("unpaked msg: %v", unpackedMsg)
 	assert.NoError(t, err)
 	assert.Equal(t, unpackedMsg.Typ, unpackerType)
 	assert.Equal(t, packers.MediaTypeZKPMessage, unpackerType)
 }
 
+func TestPackagerAnonryptPacker(t *testing.T) {
+	pm := iden3comm.NewPackageManager()
+	pm.RegisterPackers(packers.NewAnoncryptPacker(mock.ResolveEncPrivateKey), &packers.PlainMessagePacker{})
+	// nolint :
+
+	identifier := "119tqceWdRd2F6WnAyVuFQRFjK3WUXq2LorSPyG9LJ"
+
+	_, err := core.IDFromString(identifier)
+	assert.NoError(t, err)
+	var msg protocol.CredentialFetchRequestMessage
```

Comment:
Colud you build the message in one place?
like:
```go
msg := {
  "From":"...",
  "To":"..."
  etc... 
}
```

Reply:
refactored